aws/iac/dynamodb.rego

```diff
@@ -8,40 +8,37 @@ package rule
 
 default dynamodb_encrypt = null
 
-aws_attribute_absence["dynamodb_encrypt"] {
+aws_issue["dynamodb_encrypt"] {
+    resource := input.Resources[i]
+    lower(resource.Type) == "aws::dynamodb::table"
+    not resource.Properties.SSESpecification
+}
+
+aws_issue["dynamodb_encrypt"] {
     resource := input.Resources[i]
     lower(resource.Type) == "aws::dynamodb::table"
-    not resource.Properties.SSESpecification.SSEType
+    not resource.Properties.SSESpecification.SSEEnabled
 }
 
 aws_issue["dynamodb_encrypt"] {
     resource := input.Resources[i]
     lower(resource.Type) == "aws::dynamodb::table"
-    lower(resource.Properties.SSESpecification.SSEType) == "aes256"
+    resource.Properties.SSESpecification.SSEEnabled != true
```

Comment:
This is not covering string value "false" scenario. please refer [ec2.rego](https://github.com/prancer-io/prancer-compliance-test/blob/master/aws/iac/ec2.rego#L90) for checking bool values

Reply:
Fix: git show c0f6e4c

Reply:
Added string check, fixed